mixins/StoreMixin.js

```diff
@@ -76,9 +78,15 @@ var StoreMixin = {
   store_addListeners: function () {
     Object.keys(this.store_listeners).forEach(function (storeID) {
       var listenerDetail = this.store_listeners[storeID];
+      var events = listenerDetail.events;
 
+      // Check that we actually have events to fire events on
+      if (typeof events !== 'object' || !Object.keys(events).length) {
```

Comment:
Ok, done. We do test for production here: https://github.com/dcos/dcos-ui/blob/master/webpack/webpack.production.babel.js#L34

I've tested this out and it works :)